Code/Framework/AzToolsFramework/AzToolsFramework/AssetDatabase/AssetDatabaseConnection.h

```diff
@@ -72,6 +72,7 @@ namespace AzToolsFramework
             AddedStatsTable,
             ChangedSourceDependencySourceColumn,
             SplitMaterialBuilderAndMaterialAssetBuilder,
+            NewMaterialTypeBuildPipeline,
```

Comment:
Placeholder name I presume?

Reply:
This is bumping the asset database version number to force all assets to reprocess. In many cases, per-asset-builder version numbers can be incremented to reprocess assets from a specific builder. However in this case I changed the builder's job key which invalidates a bunch of data in the asset database and requires a full rebuild. The enum name is just a rough description of why we needed to rebuild the database.